doc/gcylcrc.tex

```diff
@@ -53,6 +53,16 @@ \subsubsection{use theme}
 \item {\em default:} ``default''
 \end{myitemize}
 
+\subsubsection{initial side-by-side views}
+
+Set the suite view panels initial orientation, when the GUI starts. This can be changed later using the View - Toggle views side-by-side option using the tool bar.
```

Comment:
Couple of comments:

 * Can you remove the comma in "orientation, when the"
 * Do you mean the "View" menu "Toggle views side-by-side" option or the toolbar "Toggle views side-by-side" button?
 * Could you split this over multiple lines - somewhat long here ~80 characters max would be good.